src/main/groovy/life/qbic/samplestatus/reporter/services/users/database/ConnectionProvider.groovy

```diff
@@ -0,0 +1,13 @@
+package life.qbic.samplestatus.reporter.services.users.database
+
+import java.sql.Connection
+import java.sql.SQLException
+
+/**
+ * Provides the ability to connect to a SQL ressource
+ *
+ * @since: 1.0.0
+ */
+interface ConnectionProvider {
```

Comment:
I think we should go for Hibernate here, instead of hazzling with the connection and queries manually